src/DotLiquid/Context.cs

```diff
@@ -389,6 +389,18 @@ private object Resolve(string key, bool notifyNotFound = true)
                 case "false":
                     return false;
                 case "blank":
+                    return new Symbol(o => {
+                        switch (o)
+                        {
+                            case null:
+                            case bool b when b == false:
+                            case string s when string.IsNullOrWhiteSpace(s):
+                            case IEnumerable e when !e.GetEnumerator().MoveNext():

```

Comment:
Is that meant to be the same test as existing below `(o is IEnumerable enumerableO) && !enumerableO.Cast<object>().Any()`?

Reply:
Yes, the original PR actually had this line for both. You pick which one you're more comfortable with.

Reply:
While `.Any()` is elegant, `.Cast<object>()` to get to access IEnumerable<T> extensions is outright ugly. I don't want to explore the possible unexpected effects of casting an unknown quantity to object (... boxing types?).

Funny thing is it appears that the enumerator needs disposing.

https://stackoverflow.com/questions/52600634/c-sharp-interface-ienumerable-any-without-specifying-generic-types

and

https://stackoverflow.com/questions/13459447/do-i-need-to-consider-disposing-of-any-ienumerablet-i-use

Reply:
I suggested using the IEnumerable extension of the first link, without touching IEnumerable<T>.

Reply:
Applied suggestion